datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -95,6 +95,9 @@ SELECT approx_percentile_cont(c3, 0.95, c1) FROM aggregate_test_100
 statement error DataFusion error: Error during planning: No function matches the given name and argument types 'APPROX_PERCENTILE_CONT\(Int16, Float64, Float64\)'\. You might need to add explicit type casts\.
 SELECT approx_percentile_cont(c3, 0.95, 111.1) FROM aggregate_test_100
 
+statement error DataFusion error: Error during planning: No function matches the given name and argument types 'APPROX_PERCENTILE_CONT\(Float64, Float64, Float64\)'\. You might need to add explicit type casts\.
```

Comment:
Won't this test pass on main too? Maybe the problem is that the sqllogictest framework doesn't display the hints 🤔

Reply:
Yeah that's the issue. Perhaps I should write a unit test instead? 🤔

Reply:
If you could that would be great (otherwise I worry we we may break this again during a refactor or something and not realize it)

Reply:
Added a unit test 👍